lib/bb/generate_report_request_form.rb

```diff
@@ -7,10 +7,10 @@ class GenerateReportRequestForm < Common::Form
                                 familyhealthhistory seifoodjournal healthcareproviders healthinsurance
                                 seiimmunizations labsandtests medicalevents medications militaryhealthhistory
                                 seimygoalscurrent seimygoalscompleted treatmentfacilities
-                                vitalsandreadings prescriptions medications vaallergies
+                                vitalsandreadings prescriptions vaallergies
                                 vaadmissionsanddischarges futureappointments pastappointments
                                 vademographics vaekg vaimmunizations vachemlabs vaprogressnotes
-                                vapathology vaproblemlist varadiology vahth wellness dodmilitaryservice ).freeze
+                                vapathology vaproblemlist varadiology vahth wellness dodmilitaryservice ).uniq.freeze
```

Comment:
Instead, the eligible_data_classes endpoint should maybe have uniq, or we should contact MHV to tell them to fix the problem.